### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 20

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#346] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325468-600a5726-e457-4cf7-8881-d58c8385c794.png)

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217824169-16629ef1-8fc0-4151-9d07-140c1840468d.png)
Non assicuro al 100% la correttezza di quest'orribile esercizio
